docs/docs/anatomy.md

```diff
@@ -168,22 +185,23 @@ export type Config =
 // prettier-ignore
 export namespace MessageDb {
   import AccessStrategy = MessageDB.AccessStrategy
```

Comment:
hm should AccessStrategy be prefixed too

Reply:
I'd rather the other stuff lost its prefix

```ts
DynamoStore.DynamoStoreCategory.create(....)
// vs
DynamoStore.Category.create(...)
```

Reply:
yeah, but some names are very generic (client, category, context)
and then there are other conventional names like Config, Store, Streams also in the same scope
Given the names should only really be used in the context of one of those Config binding wiring blocks, the fact they are a bit long is no real harm
Also Category would clash with Equinox.Category (and I wanted to rename ISyncContext to Context but again same concerns stop me esp given how overloaded the term Context is)

A lot of these things get close enough to files that have domain logic too - things like Category etc can easily encroach on and/or require constant disambiguation.

Can't see where I saw the argument for the long names - it was in some Azure Core guidlines or sometihng (not they should be conodered arbiters of taste for even 5 mins!)